### PR TITLE
fix: Set proper aws default region

### PR DESCRIPTION
**Pull request by werne2j**
### Description
Still had a reference to `AVP_AWS_REGION`, added aws region to readme under aws section

### Checklist
Please make sure that your PR fulfills the following requirements:
- [x] Reviewed the guidelines for contributing to this repository
- [x] The commit message follows the [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0/#summary).
- [ ] Tests for the changes have been updated
- [x] Docs have been added / updated

### Type of Change

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] New tests
- [ ] Build/CI related changes
- [x] Documentation content changes
- [ ] Other (please describe)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ metadata:
 type: Opaque
 fieldRef:
 
-  # So, <key> is NOT a placeholder 
+  # So, <key> is NOT a placeholder
   description: 'Selects a field of
     the pod: supports metadata.name,
     metadata.namespace, `metadata.labels[''<KEY>'']`,
```
Comment (jkayani) [inline README.md:97]: Not relevant to this PR at all but can you change `<key>` to `<KEY>` so the example is more clear. I got the casing wrong

**Review by jkayani — APPROVED** [review]
Makes me wonder if we should just forward all `AWS_` env variables to the AWS SDK like we do for Vault

**Comment by werne2j** [comment]
> Makes me wonder if we should just forward all AWS_ env variables to the AWS SDK like we do for Vault

I actually think it will do that already. If the aws go sdk read using `os.getenv` and you set an appropriate env var, it should get read. We could remove the code to set the region all together and just rely on env vars if you think that is a better approach

**Comment by werne2j** [comment]
For example:
https://github.com/aws/aws-sdk-go/blob/main/aws/defaults/defaults.go
https://github.com/aws/aws-sdk-go/blob/main/aws/credentials/env_provider.go

**Comment by jkayani** [comment]
Should clarify I was thinking about this: https://github.com/IBM/argocd-vault-plugin/blob/main/pkg/config/config.go#L167 where we collect configs for Vault from all the different places into Viper, and then set the appropriate Vault ones as env variables to send them to the SDK. That way we don't have to fiddle with Vault options in code. 

> We could remove the code to set the region all together and just rely on env vars if you think that is a better approach

I think that's what I'm thinking, yes. Have a loop like just for Vault, but looking for `AWS` prefixed Vault keys and just set them as env variables and let the AWS SDK handle using them

File: README.md [hunk 2]
```diff
@@ -112,12 +112,12 @@ metadata:
...
 
   # Notice, `avp.kuberenetes.io/ignore` annotation is set
-  annotations: 
+  annotations:
     avp.kuberenetes.io/ignore: "true"
 type: Opaque
 fieldRef:
 
-  # So, <key> is NOT a placeholder 
+  # So, <key> is NOT a placeholder
```
Comment (jkayani) [inline README.md:120]: I am the worst - can you get this one too?

**Review by jkayani — APPROVED** [review]
LGTM otherwise

**Comment by werne2j** [comment]
@jkayani i will create an issue to passthrough all aws env vars like vault and then merge this so we can get 1.1.0 release

**Pull request merged**